config/test.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Do you use this file?

Reply:
we do need this file, `node-config` requires the existence of a file for a test environment, it uses the `default.config` with overriding values written in test.json (which is empty, no value overriding)